pkg/infinispan/cluster.go

```diff
@@ -275,3 +241,58 @@ func (c Cluster) GetCacheManagerInfo(cacheManagerName, podName string) (map[stri
 	}
 	return info, nil
 }
+
+func (c Cluster) GetLoggers(podName string) (map[string]string, error) {
+	rsp, err, reason := c.Client.Get(podName, consts.ServerHTTPLoggersPath, nil)
+	if err := validateResponse(rsp, reason, err, "getting cluster loggers", http.StatusOK); err != nil {
+		return nil, err
+	}
+
+	defer rsp.Body.Close()
+	var loggers []Logger
+	if err := json.NewDecoder(rsp.Body).Decode(&loggers); err != nil {
+		return nil, fmt.Errorf("unable to decode: %v", err)
+	}
+	lm := make(map[string]string)
+	for _, logger := range loggers {
+		if logger.Name != "" {
+			lm[logger.Name] = logger.Level
+		}
+	}
+	return lm, nil
+}
+
+func (c Cluster) SetLogger(podName, loggerName, loggerLevel string) error {
+	path := fmt.Sprintf(consts.ServerHTTPModifyLoggerPath, loggerName, strings.ToUpper(loggerLevel))
+	rsp, err, reason := c.Client.Put(podName, path, "", nil)
+	if err := validateResponse(rsp, reason, err, "setting cluster logger", http.StatusNoContent); err != nil {
+		return err
+	}
+	return nil
+}
+
+func validateResponse(rsp *http.Response, reason string, err error, entity string, validCodes...int) error {
+	if err != nil {
+		return fmt.Errorf("unexpected error %s, stderr: %v, err: %v", entity, reason, err)
+	}
+
+	if len(validCodes) > 0 && rsp != nil {
```

Comment:
If we do:

```
	if (rsp == nil || len(validCodes) == 0) {
		return nil
	}
```

We can avoid additional nesting.

Reply:
Thanks, will fix